packages/suite/src/constants/suite/inputs.ts

```diff
@@ -1,5 +1,5 @@
 export enum MAX_LENGTH {
-    PIN = 9,
+    PIN = 50,
```

Comment:
what about FW older than 1.10.0? idk the statistics but im sure they are out there :)

Reply:
> what about FW older than 1.10.0? idk the statistics but im sure they are out there :)

Based on some recent statistics I'd estimate this is 6.4%, which is more than I'd have thought, given that 1.10.0 was released over a year ago. If the user attempts to set a PIN longer than 9 digits on old firmwares, they will see a `string overflow` error, which is not very instructive.

![image](https://user-images.githubusercontent.com/42678794/171823936-af1f0094-55a6-4614-82a5-c784d9955b46.png)

Reply:
so the question is, should we provide a fallback for that? and if so then we have 2 options:
1. (less work) handle this specific error and translate it something meaningful like "update your firmware..."
2. (more work) provide complete fallback (fw < 1.1.0 will still have max length 9)

Reply:
Updating the copy one touch, adding *the*

> The PIN is too long. Please upgrade your firmware to continue.

Reply:
Just to make it patently clear, what about "Please upgrade your firmware to enable extended PIN format"

Reply:
Looks good, thanks everyone.

![image](https://user-images.githubusercontent.com/42678794/173097624-334b8582-3e11-4bdf-84d8-582aa9a6da41.png)